app/src/pages/playground/ModelSupportedParamsFetcher.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect } from "react";
+import { graphql, useLazyLoadQuery } from "react-relay";
+
+import { usePlaygroundContext } from "@phoenix/contexts/PlaygroundContext";
+import { Mutable } from "@phoenix/typeUtils";
+
+import { ModelSupportedParamsFetcherQuery } from "./__generated__/ModelSupportedParamsFetcherQuery.graphql";
+
+/**
+ * Fetches the supported invocation parameters for a model and syncs them to the
+ * playground store instance.
+ */
+export const ModelSupportedParamsFetcher = ({
```

Comment:
yeah i don't mind this given the current structure, seems like we might need to change this whole system up at somepoint but i don't have anything in mind now, this is much better than before